src/main/java/com/powsybl/openloadflow/ac/outerloop/AcLoadFlowParameters.java

```diff
@@ -38,10 +38,12 @@
 
     private final boolean minImpedance;
 
+    private final boolean specificCompatibility;
+
     public AcLoadFlowParameters(SlackBusSelector slackBusSelector, VoltageInitializer voltageInitializer,
```

Comment:
@geofjamg It seems that I only support the specific compatibility for AC loadflow engine. Why ?

Reply:
I also added this option to DC mode